src/bitmap/iter.rs

```diff
@@ -139,6 +139,14 @@ impl FromIterator<u32> for RoaringBitmap {
     }
 }
 
+impl<'a> FromIterator<&'a u32> for RoaringBitmap {
+    fn from_iter<I: IntoIterator<Item = &'a u32>>(iterator: I) -> RoaringBitmap {
```

Comment:
Is it really necessary to specify the lifetimes here?

```suggestion
impl FromIterator<&u32> for RoaringBitmap {
    fn from_iter<I: IntoIterator<Item = &u32>>(iterator: I) -> RoaringBitmap {
```

Reply:
It doesn't work, I get:
```
error[E0637]: `&` without an explicit lifetime name cannot be used here
   --> src/bitmap/iter.rs:143:41
    |
143 |     fn from_iter<I: IntoIterator<Item = &u32>>(iterator: I) -> RoaringBitmap {
    |                                         ^ explicit lifetime name needed here
```